swarm: call out CA rotation as potentially dangerous with MKE (#16596)
docker/docs: engine/swarm/how-swarm-mode-works/pki.md

## Rotating the CA certificate

> > **Note**
> >
> > Mirantis Kubernetes Engine (formerly known as Docker UCP) provides an external
> > CA component to service the swarm. As the lifecycles of certificates in a MKE
> > cluster are controlled differently, users of MKE should not rotate the CA manually.
> > Users should contact Mirantis support if they need to rotate a certificate.
> 

Reviewer: Is this a fair rewording?

```suggestion
> **Note**
>
> Mirantis Kubernetes Engine (MKE), formerly known as Docker UCP, provides an external
> certificate manager service for the swarm. If you run swarm on MKE, you shouldn't
> rotate the CA certificates manually. Instead, contact Mirantis support if you need
> to rotate a certificate.

```

Author: Sounds better to be!
